### PR TITLE
Branch mengzhe add testing for task done undone

### LINKED ISSUE #125: Write test cases for Set Task Done/Undone feature

### PR DESCRIPTION
**Pull request by charliemoweng**
Fix #125

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/MarkTaskDoneCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        assertParseFailure(parser, MODULE_NAME_DESC_0 + VALID_STUDENT_ID_AMY + TASK_ID_DESC_0,
+                expectedMessage);
+
+        // missing task ID prefix
+        assertParseFailure(parser, MODULE_NAME_DESC_0 + STUDENT_ID_DESC_AMY + VALID_TASK_ID_0,
+                expectedMessage);
+    }
+
+    @Test
+    public void parse_invalidValueFailure() {
```
Comment (tanjoewel) [inline src/test/java/seedu/address/logic/parser/MarkTaskDoneCommandParserTest.java:69]: Should the test name be parse_invalidValue_failure()?
Same for MarkTaskUndoneCommandParserTest

**Review by tanjoewel — COMMENTED** [review]
Good job! :>

File: src/test/java/seedu/address/logic/parser/MarkTaskDoneCommandParserTest.java (hunk 1 above)
Comment (charliemoweng) [reply src/test/java/seedu/address/logic/parser/MarkTaskDoneCommandParserTest.java]: Thanks for spotting this one out! Fixed.

**Comment by felissaf** [comment]
LGTM

**Pull request merged**